jobs/e2e_node/arm/image-config-serial.yaml

```diff
@@ -0,0 +1,10 @@
+# To copy an image between projects:
+# `gcloud compute --project <to-project> disks create <image name> --image=https://www.googleapis.com/compute/v1/projects/<from-project>/global/images/<image-name>`
+# `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
+images:
+  ubuntu:
```

Comment:
Do you plan to use COS as well? We can add both configs here.

Reply:
Currently only Ubuntu is in scope, will add COS after testing separately.